src/Stratis.SmartContracts.Networks/SmartContractPosTest.cs

```diff
@@ -47,12 +47,7 @@ public SmartContractPosTest()
                 [BuriedDeployments.BIP66] = 330776
             };
 
-            var bip9Deployments = new BIP9DeploymentsArray
```

Comment:
@codingupastorm @rowandh where you planning to activate segwit 😉 
Please confirm this is change does not effect you (see also other networks)

Reply:
Looks good to me, no segwit activation planned :)